lib/PluginFindRecord/PluginFindRecordModal.js

```diff
@@ -0,0 +1,401 @@
+        </Button>
+        {isMultiSelect && (
+          <Fragment>
+            <div>
+              <FormattedMessage
+                id="stripes-smart-components.pfr.totalSelected"
+                values={{ count: checkedRecordsLength }}
+              />
+            </div>
+            <Button
+              marginBottom0
+              data-test-find-records-modal-save
+              onClick={this.saveMultiple}
+              disabled={!checkedRecordsLength}
+              buttonStyle="primary"
+            >
+              <FormattedMessage id="stripes-smart-components.pfr.button.save" />
+            </Button>
+          </Fragment>
+        )}
+      </div>
+    );
+
+    return (
+      <Modal
+        data-test-find-records-modal
+        dismissible
+        enforceFocus={false}
+        label={modalLabel}
+        onClose={closeModal}
+        open
+        contentClass={css.pluginModalContent}
+        style={{ minHeight: '500px' }}
+        size="large"
+        footer={footer}
+      >
+        <div className={css.pluginModalNewBtnWrapper}>
+          {renderNewBtn()}
+        </div>
+        <div
+          data-test-find-records
+          className={isMultiSelect ? css.showButtonsBar : ''}
+        >
+          <SearchAndSortQuery
+            querySetter={querySetter}
+            queryGetter={queryGetter}
+            onComponentWillUnmount={onComponentWillUnmount}
+            initialSearch={initialSearch}
+            initialSearchState={{ qindex: '', query: '' }}
+            syncToLocationSearch={false}
+          >
+            {
+              ({
+                searchValue,
+                getSearchHandlers,
+                onSubmitSearch,
+                onSort,
+                getFilterHandlers,
+                activeFilters,
+                filterChanged,
+                searchChanged,
+                resetAll,
+              }) => {
+                const disableReset = () => {
+                  if (filterChanged || searchChanged) {
+                    return false;
+                  }
+
+                  return true;
+                };
+
+                return (
+                  <Paneset id={`${idPrefix}-paneset`}>
+                    {this.state.filterPaneIsVisible &&
+                      <Pane
+                        defaultWidth="22%"
+                        paneTitle={<FormattedMessage id="stripes-smart-components.searchAndFilter" />}
+                      >
+                        <form onSubmit={onSubmitSearch}>
+                          <div className={css.searchGroupWrap}>
+                            <SearchField
+                              autoFocus
+                              className={css.searchField}
+                              data-test-plugin-search-input
+                              marginBottom0
+                              name="query"
+                              onChange={getSearchHandlers().query}
```

Comment:
It'd probably be useful to have an `onClear={getSearchHandlers().reset}` here.

Reply:
This code is based on ui-plugin-find-user, by @JohnC-80 . I'm not very familiar with it and I don't see why your proposal is useful, sorry. My intention is not to break things, so maybe you could put more details to this

Reply:
If you don't provide an `onClear` to `SearchField`, it doesn't show the clear icon. By providing one that clears the search, you support clearing just the search field as opposed to having to reset the entire search and filters.